dev/user.clj

```diff
@@ -242,12 +253,35 @@
 
 
   ;; Query Peer
-  (start {:fdb-query-peer-servers "localhost:8090"
-          :fdb-api-port            8080
-          :fdb-mode                "query"})
+  (start {:fdb-api-port 8080
+          :fdb-mode "query"
+          :fdb-query-peer-servers "localhost:8090"
+          :fdb-group-servers "query-server@localhost:11002"
+          :fdb-group-this-server "query-server"
+          :fdb-storage-type "memory"
+          :fdb-consensus-type "in-memory"})
 
   (stop)
 
+  (def ledger-peer (start-one {:fdb-api-port 8090
+                               :fdb-mode "ledger"
+                               :fdb-group-servers "ledger-server@localhost:11001"
+                               :fdb-group-this-server "ledger-server"
+                               :fdb-group-log-directory "./build/data/group"
+                               :fdb-storage-file-root "./build/data"
+                               #_#_:fdb-storage-type "memory"
+                               #_#_:fdb-consensus-type "in-memory"}))
+  (def query-peer (start-one {:fdb-api-port 8080
```

Comment:
should we change this port to something other than 8080, for the same reasons we moved the ledger port to 8090?

Reply:
This is just for local development so I don't think it's too big of a deal, but I can change it just to be safe.